example_app_generator/generate_app.rb

```diff
@@ -21,54 +21,37 @@
   gsub_file 'Gemfile', /^.*\bgem 'rails.*$/, ''
   gsub_file "Gemfile", /.*web-console.*/, ''
   gsub_file "Gemfile", /.*debugger.*/, ''
-  gsub_file "Gemfile", /.*byebug.*/, "gem 'byebug', '~> 9.0.6'"
-  gsub_file "Gemfile", /.*puma.*/, ""
-  gsub_file "Gemfile", /.*gem..sqlite3.*/, "gem 'sqlite3', '~> 1.3.6'"
-  gsub_file "Gemfile", /.*bootsnap.*/, ""
-
-  if Rails::VERSION::STRING >= '5.0.0'
-    append_to_file('Gemfile', "gem 'rails-controller-testing'\n")
-  end
```

Comment:
This should stay due to the "soft" 4.2 support

Reply:
Nevermind, I see we're now adding it for all.

Reply:
You are actually right here. In our builds - sure, we don't need that condition. But for whoever decides to debug why RSpec doesn't work with their Rails 4.2 that will be a problem since `rails-controller-testing` [depends on Rails 5+](https://github.com/rails/rails-controller-testing/blob/v1.0.0/rails-controller-testing.gemspec#L21) (this is the first time I see `~> 5.x` syntax).

Going to revert this removal and add a comment.

Reply:
Are you going to revert this?

Reply:
Oops, moved it [down below](https://github.com/rspec/rspec-rails/pull/2380/files#diff-a8a7f2d3f06e0c6f9a7717dd8c7cb715R37). Let me quickly move it back.

Reply:
Done.